hack/components/bump-linux-bridge.sh

```diff
@@ -43,6 +43,8 @@ RUN mkdir -p ${LINUX_BRIDGE_TAR_CONTAINER_DIR}
 RUN microdnf install -y findutils
 COPY --from=builder ${LINUX_BRIDGE_PATH}/bin/bridge ${LINUX_BRIDGE_TAR_CONTAINER_DIR}/bridge
```

Comment:
can the COPY itself copy directly to the needed folder ? or its not a good idea ?